spyderlib/spyder.py

```diff
@@ -1144,7 +1146,11 @@ def add_xydoc(text, pathlist):
                                          self.close_dockwidget_action,
                                          self.maximize_action,
                                          None))
+            self.toggle_toolbars_action = create_action(self,
+                                    _("Toggle toolbar visibility"),
+                                    toggled=self.toggle_toolbars)

```

Comment:
This method (`self.toggle_toolbars`) and the corresponding action should be renamed if you decide to follow my suggestion.